crates/rolldown_binding/src/options/binding_output_options/mod.rs

```diff
@@ -20,7 +20,7 @@ pub struct BindingOutputOptions {
   // assetFileNames: string | ((chunkInfo: PreRenderedAsset) => string);
   #[derivative(Debug = "ignore")]
   #[serde(skip_deserializing)]
-  #[napi(ts_type = "undefined | string | ((chunk: RenderedChunk) => string | Promise<String>)")]
+  #[napi(ts_type = "VoidNullable<string> | ((chunk: RenderedChunk) => MaybePromise<VoidNullable<string>>)")]
```

Comment:
```suggestion
  #[napi(ts_type = "Nullable<string> | ((chunk: RenderedChunk) => MaybePromise<VoidNullable<string>>)")]
```

You also need to run `pnpm build` to update generate types. You could run `just roll-node` to check if it's correct.

Reply:
`just fmt` will fix almost every lint errors.